### ISSUE TITLE
[BUG]: Accessibility issue - Flashing header

### ISSUE THREAD

**Issue by laflannery** [CivicActions/civicactions-homesite#284]
**Expected behavior**
There should be no flashing or flickering movement on the site. This is a violation of WCAG 2.3.1 and could cause seizures in certain users.

**Describe the bug**
On a page that has the red swooshy background image in the header, at a certain point when you are scrolling an the image is about to disappear, it starts to flash quite quickly.

**To Reproduce**
List the steps to reproduce the bug:
1. Go to [Press](https://civicactions.com/press/)
2. Start to scroll down the page
3. When you get to the point where the H1 is right at the top of the page, you should start to see the red background image flash over in the left corner

**Add Screenshots**
This is a screenshot of how the screen looks, where the H1 is when the flickering takes place, I'm going to try and get a video as well

![Image](https://github.com/user-attachments/assets/58a09394-4688-4b77-a735-919ac2d935b8)

When it's flashing, the `scrolled-mobile` class on the `<header>` here is also flashing (if that helps to resolve):

![Image](https://github.com/user-attachments/assets/94be0f16-43e5-4fd4-b4c7-2a456bb4a88e)

**Desktop (what device and system were you using?):**
 - Browser: Chrome

**Smartphone (what device and system were you using?):**

**Additional context**
Adrian happened to bring this up during the Practice area meeting and both Jordan and I were able to reproduce it
